packages/ckeditor5-html-support/docs/features/general-html-support.md

```diff
@@ -214,6 +214,21 @@ The General HTML Support feature distinguishes several content types, each treat
 
 The enabled elements will not just be available "anywhere" in the content, as they still need to adhere to certain rules derived from the HTML schema and from common sense. Also, the behavior of specific types of elements in the editing area will be different. For instance, the object elements will only be selectable as a whole, and the inline elements will work the same as other formatting features supported by CKEditor 5 (e.g. bold, italic) do.
 
+### Enabling all elements and attributes
+
+In some cases, it might be desired to disable content filtering, so all elements and attributes will be allowed. It could be done with a special configuration:
+
+```js
+{
+	name: /.*/,
+	attributes: true,
+	classes: true,
+	styles: true
+}
+```
+
+The above configuration will work similarly to `allowedContent: true` option from CKEditor 4.
```

Comment:
Do you think it makes sense to link to https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-allowedContent when we are mentioning the CKE4 feature? Not sure if we want to do it from CKE5 docs but it would improve navigation for someone migrating content from CKE4.

Reply:
Yeah, I was thinking about it and wasn't sure if it makes sense to do that. I'll add a link :+1: